k8s/overlays/coreso/prod/postgresql-configmap.yaml

```diff
@@ -38,6 +38,10 @@ data:
     CREATE DATABASE swe_idcc_tasks;
     CREATE USER $POSTGRES_SWE_IDCC_USER WITH ENCRYPTED PASSWORD '$POSTGRES_SWE_IDCC_PASSWORD';
     GRANT ALL PRIVILEGES ON DATABASE swe_idcc_tasks TO $POSTGRES_SWE_IDCC_USER;
+    CREATE DATABASE swe_idcc_idcf_tasks;
+    CREATE USER $POSTGRES_SWE_IDCC_IDCF_USER WITH ENCRYPTED PASSWORD '$POSTGRES_SWE_IDCC_IDCF_PASSWORD';
+    GRANT ALL PRIVILEGES ON DATABASE swe_idcc_idcf_tasks TO $POSTGRES_SWE_IDCC_IDCF_USER;
+    CREATE DATABASE cse_import_ec_idcc_tasks;
```

Comment:
I believe this line has been added by mistake since it is already present in the file, line 32

Reply:
well seen, thanks